docs/advanced/classes.rst

```diff
@@ -784,6 +788,36 @@ memory corruption and/or segmentation faults.
 
 .. [#f3] http://docs.python.org/3/library/pickle.html#pickling-class-instances
 
+Deepcopy support
+================
+
+Python normally uses references in assignments. Sometimes a real copy is needed
+to prevent changing all copies. The ``copy`` module [#f5]_ provides these
+capabilities.
+
+The ``__setstate__`` and ``__getstate__`` methods provided to support pickle,
```

Comment:
I was slightly thrown off track by this level of detail. How about:
A class with pickle support is automatically also (deep)copy compatible. However, performance can be improved ...